compiler/noirc_frontend/src/lexer/token.rs

```diff
@@ -3,11 +3,11 @@ use noirc_errors::{Position, Span, Spanned};
 use std::{fmt, iter::Map, vec::IntoIter};
 
 use crate::{
-    lexer::errors::LexerErrorKind,
+    ast::Expression, lexer::errors::LexerErrorKind,
     node_interner::{
         ExprId, InternedExpressionKind, InternedPattern, InternedStatementKind,
         InternedUnresolvedTypeData, QuotedTypeId,
-    },
+    }
```

Comment:
Have you run `cargo fmt` on this?

Reply:
No, I haven't. Thanks for the fine observation.
Now I did and I pushed the changes.